.github/workflows/checks.yml

```diff
@@ -125,6 +125,13 @@ jobs:
         working-directory: build
       - name: Set up problem matcher
         uses: ammaraskar/gcc-problem-matcher@0.3.0
+      # Work around https://github.com/actions/runner-images/issues/8659
+      - name: "Remove GCC 13 from runner image (workaround)"
+        shell: bash
+        run: |
+          sudo rm -f /etc/apt/sources.list.d/ubuntu-toolchain-r-ubuntu-test-jammy.list
+          sudo apt-get update
+          sudo apt-get install -y --allow-downgrades libc6=2.35* libc6-dev=2.35* libstdc++6=12.3.0* libgcc-s1=12.3.0*
```

Comment:
[Coverage fails at this step](https://github.com/mixxxdj/mixxx/actions/runs/17711787499/job/50331410994?pr=15359)
@daschuer Is that only relevant for 2.5 targets and can be removed?

Reply:
Yes, let me fix this!